data/src/test/java/org/apache/iceberg/data/TestMetricsRowGroupFilter.java

```diff
@@ -311,6 +313,14 @@ public void testIsNaN() {
     Assert.assertFalse("Should skip: all null column will not contain nan value", shouldRead);
   }
 
+  @Test
+  public void testIsNaNForORC() {
+    Assume.assumeFalse("Not for Parquet files", format == FileFormat.PARQUET);
```

Comment:
Same note about the `Assume` statement.

I would just put `Assume.assumeTrue(format == FileFormat.ORC)` unless you think that Avro should also be included here.

Many of the `Assume` statements don't seem to give a reason phrase at all.